### PR TITLE
Task4 filter(L4)

### PR DESCRIPTION
**Pull request by wi9916**

### CODE REVIEW

**Comment by wi9916** [comment]
[HttpGet("GetAllFoods")] нужен для свагера

File: Web/RockFood.Api/Filter/FoodActionFilter.cs [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+                {
+                    stringContent = await reader.ReadToEndAsync();
+                    context.HttpContext.Request.Body.Position = 0;
+                }
+
+                rContext = await next();               
+            }
+            catch (Exception)
+            {
+                throw;
```
Comment (yevhenbilushko) [inline Web/RockFood.Api/Filter/FoodActionFilter.cs]: log error or remove try catch
as for me it has no sense to catch and throw

File: Web/RockFood.Api/App_Code/ListHelper.cs [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+        {
+            var result = "<tbody>";
+            foreach (var food in foods)
+            {
+                result = $"<tr>{result}<td>{food.Name}</td>";
+                result = $"{result}<td>{food.About}</td>";
+                result = $"{result}<td>{food.Price}</td>";
+                result = $"{result}<td>{food.Count}</td>";
+                result = $"{result}<td><a href=\"/FoodMVC/Edit/{food.Id}\"> Edit </a> | ";
+                result = $"{result}<a href=\"/FoodMVC/Details/{food.Id}\"> Details </a> | ";
```
Comment (yevhenbilushko) [inline Web/RockFood.Api/App_Code/ListHelper.cs]: it is cool that u do more additional functionality here
I mean links

but sense is to reuse this helper for products, cities, cars and so on
and cars know nothing about FoodMVC

so I mean that lets remove specific things and make it for general use

**Review by yevhenbilushko — DISMISSED** [review]
fix and merge

**Pull request merged**